Gemfile

```diff
@@ -21,6 +21,8 @@ gem "minima", "~> 2.0"
 group :jekyll_plugins do
   gem "jekyll-feed", "~> 0.17"
   gem 'jekyll-asciidoc'
+  gem 'asciidoctor'
```

Comment:
Do we actually need to explicitly specify `asciidoctor` in `Gemfile`? Does it not just get pulled in automatically by `asciidoctor-tabs` ?

Reply:
Something about jekyll-asciidoc breaks extension support if you don't explicitly include asciidoctor as a gem here. I tried to work around this for quite some time but eventually just caved and included it here.